command/job_stop.go

```diff
@@ -140,92 +140,141 @@ func (c *JobStopCommand) Run(args []string) int {
 		return 1
 	}
 
-	// Check if the job exists
-	jobs, _, err := client.Jobs().PrefixList(jobID)
-	if err != nil {
-		c.Ui.Error(fmt.Sprintf("Error deregistering job: %s", err))
-		return 1
-	}
-	if len(jobs) == 0 {
-		c.Ui.Error(fmt.Sprintf("No job(s) with prefix or id %q found", jobID))
-		return 1
+	// TODO: since we're printing errors as
+	// we receive them, we don't need a new ch
+	// errorCh := make(chan error, len(jobIDs))
+
+	statusCh := make(chan int, len(jobIDs))
+
+	var wg sync.WaitGroup
+	for _, jobID := range jobIDs {
+		jobID := jobID
+
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+
+			// Truncate the id unless full length is requested
+			length := shortId
+			if verbose {
+				length = fullId
+			}
+
+			// Check if the job exists
+			jobs, _, err := client.Jobs().PrefixList(jobID)
+			if err != nil {
+				// TODO: can we improve this error message?
+				c.Ui.Error(fmt.Sprintf("Error deregistering job with id %s err: %s", jobID, err))
```

Comment:
Should this say `Error finding jobs with prefix: <>` instead?

Reply:
Yes.